app/utils/theme.py

```diff
@@ -0,0 +1,89 @@
+from pathlib import Path
+from typing import Optional
+
+from loguru import logger
+
+from app.utils.app_info import AppInfo
+
+theme_data_folder = AppInfo().theme_data_folder
+theme_storage_folder = AppInfo().theme_storage_folder
```

Comment:
I'm not a fan of doing global constants for these. 

I'd have these just be class attributes passed at init that you can override, and then decouple the entire class from AppInfo.

Reply:
i have updated hope u agree with current changes